tool/android_ci_script.sh

```diff
@@ -37,7 +37,7 @@ do
     echo "== Testing '${PROJECT_NAME}' on Flutter's stable channel =="
     pushd "${PROJECT_NAME}"
 
-    gradle wrapper
```

Comment:
This call fails also on my computer, with the error:

```

FAILURE: Build failed with an exception.

* Where:
Script '/home/miquel/dev/tools/flutter/packages/flutter_tools/gradle/module_plugin_loader.gradle' line: 59

* What went wrong:
A problem occurred configuring root project 'AndroidFullScreen'.
> A problem occurred configuring project ':flutter'.
   > Failed to notify project evaluation listener.
      > org/gradle/api/plugins/MavenPlugin
```

This call seems to be unnecessary, since the gradlew files already exist in the repo.

Trying to run the CI script without it, see if it works.

Reply:
Looks like this problem was caused by using a very old version of gradle in the gradle.build of the project, trying to upgrade it now

Reply:
Back to this question: I removed the call to `gradle wrapper` and instead decided to include the gradlew files in the repo.

I think this is the recommended way of distributing Android code, the idea is that everyone uses the same version of gradle embedded in the gradlew wrapper instead of generating a new one.

Secondly, this command was causing problems because the gradle version found in the CI server is 8.1, which is not fully compatible with Flutter. All the work I have done locally is using the recommended gradle 7.2.